client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -79,7 +79,7 @@ interface EditorProps {
   canFocus: boolean;
   challengeFiles: ChallengeFiles;
   challengeType: number;
-  containerRef: RefObject<HTMLElement>;
+  containerRef: RefObject<HTMLElement | undefined>;
```

Comment:
There is `RefObject` import that can be deleted as well, and `editorRef` type below dimensions that should be updated to this `editorRef: MutableRefObject<editor.IStandaloneCodeEditor| undefined>;`

```suggestion
  containerRef: MutableRefObject<HTMLElement | undefined>;
```